cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml

```diff
@@ -158,7 +158,11 @@ components:
                             PATO:
                                 - PATO:0000461
                             MONDO:
-                                - all
+                                - MONDO:0021178
+                        ancestors:
```

Comment:
> child(ren) terms

Shouldn't this be `descendants`? The `_validate_curie` function would require some updates...

Reply:
Ah, I see how this is being used. Any term that has one of the `allowed` `ancestors` is itself allowed. Interesting. Slightly nonintuitive 😅